diagnostic.md

```diff
@@ -16,13 +16,13 @@ Graces-MacBook-Pro:try grace$
 
 What is the full path to the current directory?
 
-_Replace this text with your answer._
+/Users/grace/wdi/try
 
 ## Question 2
 
 Describe GitHub, **briefly** in your own words.
 
-_Replace this text with your answer._
+Github is an open-sourced, collaborative online hosting site that allows developers to host their own projects and share the 'inner-workings' (aka code) with others.
```

Comment:
Correct. In a way, it's a User Interface for git and version control of our repositories.